assets/onboarding/features/installation-feedback.jsx

```diff
@@ -1,49 +1,79 @@
+import { useState, useEffect } from '@wordpress/element';
 import { List } from '@woocommerce/components';
 import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
 import FeatureDescription from './feature-description';
-import FeatureStatus, { LOADING_STATUS, ERROR_STATUS } from './feature-status';
+import FeatureStatus, {
+	INSTALLING_STATUS,
+	ERROR_STATUS,
+} from './feature-status';
+import useFeaturesPolling from './use-features-polling';
+
+const getStatus = ( status = INSTALLING_STATUS ) => status;
```

Comment:
Is it needed to set installing as the default status?

Reply:
We needed it because the features remained without status until the first status request is completed.

But as we created a reducer to clear the errors for the retries, I used the same to set the `installing` status, which make much more sense =) https://github.com/Automattic/sensei/pull/3190/commits/d72b32c5c7c6991b8558ac0191180b224f310bf7

Also, when I updated the approach I found a cache issue, fixed here: https://github.com/Automattic/sensei/pull/3190/commits/742808427af9f40e4f29ad124164471884fc1ac1

Thank you for the question!